SIP-X.md

```diff
@@ -0,0 +1,75 @@
+|                   |	LoanClosingsWith            | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanMaintenance             | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanOpenings                | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	LoanSettings                | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	ProtocolSettings            | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	SwapsExternal               | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+| Other             |                             |       |                                                              |
+|                   |	Liquidity Mining            | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	StakingRewards              | Owner	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	VestingLogic                | Owner	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	Modules                     | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+| Connectors        |                             |       |                                                              |
+|                   |	LoanToken                   | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	LoanTokenLogicBeacon        | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	LoanTokenSettingsLowerAdmin | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	LoanTokenLogicStandard      | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+| Core              |	                            |     	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	Protocol                    | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+| Oracles           |	                            |     	|                                                              |
+|                   |	BPro Price Feed             | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	RSK Price Feed              | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	MoC Price Feed              | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	Price Feeds Gateway         | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	PriceFeedV1PoolOracle       | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
```

Comment:
note: on the contracts there is only the role `owner`, but the owner has only administrative power, so it should be fine to place it under GovernorAdmin.

Reply:
Well, that is proper confusing xD thanks for letting me know